service/web/adminserver_test.go

```diff
@@ -247,3 +249,177 @@ func TestAdminServer_ListProjectCredentials(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expectedCredProto, p)
 }
+
+func Test_adminServer_CreateProjectCredentials(t *testing.T) {
+	ctx := context.Background()
+	ctrl := gomock.NewController(t)
+	mockCredentialsCore := mocks4.NewMockICore(ctrl)
+	req := &metrov1.ProjectCredentials{ProjectId: "test-project"}
+
+	tests := []struct{ err error }{
+		{err: nil},
+		{err: fmt.Errorf("Something went wrong")},
+	}
+	for _, test := range tests {
+		mockCredentialsCore.EXPECT().Create(ctx, gomock.Any()).Times(1).Return(test.err)
+		adminServer := newAdminServer(
+			&credentials.Model{Username: "u", Password: "p"},
+			mocks.NewMockICore(ctrl),
+			mocks2.NewMockICore(ctrl),
+			mocks3.NewMockICore(ctrl),
+			mockCredentialsCore,
+			mocksnb.NewMockICore(ctrl),
+			nil,
+		)
+		got, err := adminServer.CreateProjectCredentials(ctx, req)
+		assert.Equal(t, test.err != nil, err != nil)
+		assert.Equal(t, test.err == nil, got != nil)
+	}
+}
+
+func Test_adminServer_DeleteProjectCredentials(t *testing.T) {
+	ctx := context.Background()
+	ctrl := gomock.NewController(t)
+	mockCredentialsCore := mocks4.NewMockICore(ctrl)
+	credentialModel := getDummyCredentialModel()
+
+	req := &metrov1.ProjectCredentials{
+		ProjectId: "test-project",
+		Username:  "test-project__00f790",
+	}
+
+	tests := []struct {
```

Comment:
you include name of test in test struct and run test case like for _, tt := range tests { t.Run(tt.name, func(t *testing.T) {}}
this will log test name if test case is failing